extensions/rich_text_components/Image/directives/image.component.html

```diff
@@ -1,11 +1,19 @@
 <figure>
-  <div *ngIf="!isLoadingIndicatorShown && !isTryAgainShown">
+  <div *ngIf="!isLoadingIndicatorShown && !isTryAgainShown && !showAltText">
     <img [src]="imageUrl"
          alt="{{imageAltText}}"
          class="center-element-horizontally e2e-test-image"
          [ngStyle]="imageContainerStyle">
     <figcaption class="text-center">{{imageCaption}}</figcaption>
   </div>
+  <div *ngIf="!isLoadingIndicatorShown && !isTryAgainShown && showAltText">
+    <img [src]="imageUrl"
+         alt="{{imageAltText}}"
+         class="e2e-test-image mt-5"
+         [ngStyle]="imageContainerStyle">
+    <figcaption><strong>Caption:</strong> {{imageCaption}}</figcaption>
+    <figcaption class="alt-text mb-5"><strong>Description:</strong> {{imageAltText}}</figcaption>
+  </div>
```

Comment:
Could you check if the image component works the same way it used to in the exploration editor page? Please add a video in the PR description for the same.

Reply:
Done, PTAL